### PR TITLE
Add t3 instance types

### PR DESCRIPTION
**Pull request by andresbono**
*Issue #, if available:*

#35 

*Description of changes:*

This PR adds the `t3` instance type family to the `AllowedValues`.

This PR depends on https://github.com/aws-quickstart/quickstart-linux-bastion/pull/65.

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/rdsaurora.template [hunk 1]
```diff
@@ -43,6 +43,12 @@
                 "db.t2.large",
                 "db.t2.xlarge",
                 "db.t2.2xlarge",
+                "db.t3.micro",
+                "db.t3.small",
+                "db.t3.medium",
```
Comment (Erowlin) [inline templates/rdsaurora.template]: 🔧  For Aurora, only `t3.small` and `t3.medium` are available to me. t3.micro and t3.large / xlarge and 2xlarge are not listed. 
image

Comment (Erowlin) [reply templates/rdsaurora.template]: BTW, other values in this file may be wrong - Like `db.t2.micro` doesn't exist to me.

File: templates/wordpress-master.template [hunk 2]
```diff
@@ -271,6 +277,12 @@
                 "db.t2.large",
                 "db.t2.xlarge",
                 "db.t2.2xlarge",
+                "db.t3.micro",
+                "db.t3.small",
+                "db.t3.medium",
+                "db.t3.large",
+                "db.t3.xlarge",
+                "db.t3.2xlarge",
```
Comment (Erowlin) [inline templates/wordpress-master.template]: 🔧  Same remark : only t3.small and t3.medium are available to me.

**Comment by andrew-glenn** [comment]
@Erowlin - It's a catch-22. Generally the PR supports all *supported* db.t3 instance types. However, not all instance types are *available* in all AZs.

**Pull request merged**